### ISSUE TITLE
Support For FireForx & Safari + IOS Devices

### ISSUE THREAD

**Issue by danishs360** [livekit/track-processors-js#38]
Is there a support for firefox and safari browsers + IOS

Because its failing by returning : processor is not supported by the browseer

**Comment by danishs360** [comment]
hmm thanks, any workaround for that? its very crucial for us to have background blur with livekit :)

**Comment by lukasIO** [comment]
not that I know of, sorry

**Comment by awmichel** [comment]
@lukasIO Have you looked into Encoded Transforms? The docs read like this is the API that Firefox and Safari implemented that should be equivalent-ish to Insertable Streams, but I haven't fully researched and compared.

https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Using_Encoded_Transforms

**Comment by lukasIO** [comment]
yeah, we use that for cross browser e2ee, but there are different constraints for the track processors ( https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrackProcessor)

**Comment by SachinNumly** [comment]
Any updates on the issue to support on iOS and Mac - Safari?
